tests/x509/test_x509.py

```diff
@@ -3615,8 +3616,11 @@ def test_build_ca_request_with_ed25519(self, backend):
         assert list(subject) == [
             x509.NameAttribute(NameOID.STATE_OR_PROVINCE_NAME, "Texas"),
         ]
-        basic_constraints = request.extensions.get_extension_for_oid(
-            ExtensionOID.BASIC_CONSTRAINTS
+        basic_constraints = typing.cast(
+            "x509.Extension[x509.BasicConstraints]",
```

Comment:
Can this be a non-string, or is that an error?

Reply:
apparently it can.